stylesOLD.css

```diff
@@ -0,0 +1,105 @@
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: inherit;
+  -moz-box-sizing: inherit;
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: blanchedalmond;
+  font-family: 'Cocogoose', sans-serif;
+  padding: 0 20px;
+}
+
+.menu-on {
+  overflow: hidden;
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5));
+}
+
+/* .br {
+  display: block;
+  margin-bottom: 0em;
+}
+
+.brmedium {
+  display: block;
+  margin-bottom: 1em;
+}
+
+.brlarge {
+  display: block;
+  margin-bottom: 2em;
+} */
+
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code bulky and untidy